src/bridge/index.js

```diff
@@ -102,18 +81,33 @@ const extractDocMetadata = () => {
       }
     })
     const { title, description, url } = graphData
-    return { title, text: description, url }
+    const openGraphMetadata = filterObjectFromNullValues({
+      title,
+      text: description,
+      url,
+    })
+    metadata = { ...metadata, ...openGraphMetadata }
```

Comment:
If we're trying to be precise, these merges should ideally be handled by a `deep-merge-right` type of mechanism. In the interest of time (and not currently realistically handling metadata objects with a higher depth than 1), I will leave it as is for now.